.github/workflows/daily.yml

```diff
@@ -0,0 +1,64 @@
+name: Daily runs
+on:
+  schedule:
+    - cron: '0 5 * * *'
```

Comment:
Please also run this workflow on push when there are changes to `.github/workflows/daily.yml`.